src/celengine/mapmanager.h

```diff
@@ -22,7 +24,13 @@ class WarpMesh
     ~WarpMesh();
 
     // Map data to triangle vertices used for drawing
-    void scopedDataForRendering(const std::function<void(float*, int)>&) const;
+    template<typename T>
+    void scopedDataForRendering(T func) const
```

Comment:
maybe better to remove this function and where it's used replace with direct usage of getScopedData()

Reply:
done